project_template/static/src/js/modifiers.js

```diff
@@ -0,0 +1,69 @@
+/*
+    © 2019 Numigi (tm) and all its contributors (https://bit.ly/numigiens)
+    License LGPL-3.0 or later (http://www.gnu.org/licenses/LGPL.html).
+*/
+odoo.define("project_template.hidden_fields", function(require) {
+"use strict";
+
+var Domain = require('web.Domain');
+
+function isNodeInvisibleOnTemplate(node){
+    var invisibleAttribute = node.attrs.invisible_on_template;
+    return invisibleAttribute && Boolean(eval(node.attrs.invisible_on_template));
+}
+
+function isProjectOrTask(state){
+    var model = state.model;
+    return model == "project.project" || model == "project.task";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected '===' and instead saw '=='. (eqeqeq)](https://app.codacy.com/app/numigi/odoo-project-addons/pullRequest?prid=4397703)